Custom-password-dialog-box/MainWindow.xaml.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.Win32;
+using System;
+using System.IO;
+using System.Security;
+using System.Windows;
+
+namespace CustomPasswordDialog
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        string preLoadPDF;
+        PasswordDialog passwordDialog;
+        public MainWindow()
+        {
+            InitializeComponent();
+        }
+        private void Window_Loaded(object sender, RoutedEventArgs e)
+        {
+            pdfViewer.Visibility = Visibility.Hidden;
```

Comment:
Why we are hiding the PdfViewer unwantedly? Also remove the open icon from default toolbar